src/core/Akka/Actor/TypedActor.cs

```diff
@@ -25,6 +26,7 @@ public interface IHandle<in TMessage>
     /// <summary>
     ///     Class TypedActor.
     /// </summary>
+    [Obsolete("TypedActor in its current shape will be removed. Use ReceiveActor instead")]
```

Comment:
ReceiveActor is the default generic actor type most people are using. ReceiveActor has been the default actor type we have been recommending for a while now. It only makes sense to put that in here as well.